### PR TITLE
Docs: SAML refactoring docs

### PR DESCRIPTION
**Pull request by eleijonmarck**
**What this PR does / why we need it**:

This moves the unclear part of the enable saml to troubleshooting where it should be.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/enterprise/saml/troubleshoot-saml.md [hunk 1]
```diff
@@ -26,6 +26,42 @@ We only support one private key format: PKCS#8.
...
+This creates a new pkcs8 key file.
+
 ```bash
-$ openssl pkcs8 -topk8 -nocrypt -in <yourkey> -out private.pem
+$ openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes​
+```
+
+#### **Convert** the private key format to base64
+
+This converts already created keys to base64
```
Comment (chri2547) [inline docs/sources/enterprise/saml/troubleshoot-saml.md:37]:
```suggestion
The following command converts keys to base64 format.
```

File: docs/sources/enterprise/saml/troubleshoot-saml.md [hunk 2]
```diff
@@ -26,6 +26,42 @@ We only support one private key format: PKCS#8.
...
+It should look like:
+
+```
+-----BEGIN PRIVATE KEY-----
+...
+...
+-----END PRIVATE KEY-----
+```
+
+If you have a key that looks like:
```
Comment (chri2547) [inline docs/sources/enterprise/saml/troubleshoot-saml.md:60]: This confuses me. If I have a key that looks like the sample you provide, then what? The 'If' makes me feel like there is something missing here.

**Review by chri2547 — COMMENTED** [review]
See feedback, thank you for turning this around so quickly!

File: docs/sources/enterprise/saml/troubleshoot-saml.md (hunk 2 above)
Comment (eleijonmarck) [reply docs/sources/enterprise/saml/troubleshoot-saml.md:60]: Hmm il have a look at how I could rephrase this better

**Review by chri2547 — APPROVED** [review]

**Pull request merged**